app/components/templates/Outline.tsx

```diff
@@ -36,13 +38,13 @@ const Outline: React.SFC<{ children: ReactChild; classes: any }> = ({
             Shentaria
           </Typography>
         </Link>
-        {links.map(({ label, href }) => (
-          <Link key={label} href={href}>
-            <Typography variant="h6" color="inherit" className={classes.item}>
-              {label}
-            </Typography>
-          </Link>
-        ))}
+        <Tabs value={false}>
+          {links.map(({ label, href }) => (
```

Comment:
Tabs encapsulates link and passes props to it, using tabs here because the tab element gives us nice styling and free animations can change this in the future.

Reply:
I don't understand what the prop validation error is, do you have an example? If it's about children it may be as simple as a case of wrapping the `{links...}` block in `React.Fragment`.